qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -147,7 +147,7 @@ public void test041AmazonCaCertsAreInTheKeystore() {
     public void test042KeystorePermissionsAreCorrect() throws Exception {
         waitForElasticsearch(installation);
 
-        assertPermissionsAndOwnership(installation.config("elasticsearch.keystore"), p660);
+        assertPermissionsAndOwnership(installation.config("elasticsearch.keystore"), "elasticsearch", "root", p660);
```

Comment:
This seems backwards. Shouldn't the group be elasticsearch and the owner be root?

Reply:
The startup process creates this if necessary at startup, so it has to be `elasticsearch:root`.